### PR TITLE
fix(curriculum): added missing information for event flyer lab

### LINKED ISSUE #56874: [ Full Stack Cert ] - update demo and add missing information for event flyer lab
There are a few things that need to be updated in the lab.

The first thing is that the message for reminding campers to link their CSS to the HTML document is missing.

Here is the massage we use

```md
**Note:** Be sure to link your stylesheet in your HTML and apply your CSS.
```

This message needs to go just before the `# --hints--`

Then in the `# --solutions--` section, the height seems off here

Image

Using `min-height` for the `body` selector resolves that

```css
body {
  padding: 50px 10px;
  margin: 0 auto;
  width: 90vw;
  min-height: calc(100vh - 100px); 
  background-color: white;
  font-family: sans-serif;
  text-align: center;
}

```

Here is the file you need to update

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md

### PR DESCRIPTION
**Pull request by kamalsinghb**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56874

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:1]: Hint and test checking height needs to be updated as well.
https://github.com/freeCodeCamp/freeCodeCamp/blob/099fe1d66244ad207817113099c71f74135c3988/curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md?plain=1#L120-L124

**Comment by kamalxdev** [comment]
Changes Updated

**Review by majestic-owl448 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md [hunk 1]
```diff
@@ -115,10 +117,10 @@ Your `body` element should set its width using `vw`.
 assert(new __helpers.CSSHelp(document).getStyle('body')?.width?.endsWith('vw'));
 ```
 
-Your `body` should use `calc` to set its height to `100vh - 100px`.
+Your `body` should use `calc` to set its min-height to `100vh - 100px`.
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:120]:
```suggestion
Your `body` should use `calc` to set its `min-height` to `100vh - 100px`.
```
Hmm, I think this should be in backticks...

Comment (kamalxdev) [reply curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md]: done

**Review by gikf — APPROVED** [review]

**Pull request merged**